### ISSUE TITLE
It is better to develop ahk+ and ahk++ separately.

### ISSUE THREAD

**Issue by cweijan** [vscode-autohotkey/.github#4]
After I checked the latest code, I found that a lot of tools were introduced in it, but I hope the project can be kept simple.

Especially Prettier, I think its formatted code is ugly. It always wraps parameter definition and parameter passing, I can't accept it.
![image](https://user-images.githubusercontent.com/27798227/203502649-fce98b92-65ee-44ce-a899-39f0194aaf05.png)

We don't need to merge the two projects. @mark-wiemer

**Comment by mark-wiemer** [comment]
Yeah, I'm OK with that :) I'll open PRs on AHK+ to add new features that are added to AHK++. We can keep the projects separate but both in this organization for consistency, if that's OK. It'll also make it easier to share issues across the two repos

**Comment by cweijan** [comment]
Thank you for your understanding, I agree with this.
